examples/elasticsearch-incremental/elasticsearch-incremental-blueprint.yaml

```diff
@@ -0,0 +1,542 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata: 
+  name: elasticsearch-incremental-blueprint 
+actions:
+  backup:
+    outputArtifacts:
+      s3Snap:
+        keyValue:
+          repoPath: '{{ .Phases.setupPhase.Output.repoPath }}'
+          snapshotName: '{{ .Phases.setupPhase.Output.snapshotName }}'
+          esClusterName: '{{ .Phases.setupPhase.Output.esClusterName }}'
+          esClusterNamespace: '{{ .Object.metadata.namespace }}'
+    phases:   
+    - func: KubeTask
+      name: setupPhase
+      objects:
+        elasticSecret:
+          kind: Secret
+          name: '{{ .Object.metadata.name }}-es-elastic-user'
+          namespace: '{{ .Object.metadata.namespace }}' 
+      args:        
+        image: ghcr.io/kanisterio/kanister-kubectl-1.18:0.81.0
+        command:
+        - bash
+        - -x
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          debug=0
+          function dbg
+          {
+              if [[ $debug -eq 1 ]]
+              then 
+                echo "$(date "+%Y-%m-%d %H:%M:%S.%3N") - $0 - $*" >> /tmp/k10_debug.log
+              fi
```

Comment:
I am assuming this logs file is created inside the pod that gets created by KubeTask pod? In which cases it's useful? Is this useful if the pod fails because of certain reason? If yes, would be get the file that was created by earlier run?

Reply:
No specifically this one is for kubeExec, for kubepod I prefer the one you commented 
```
function bp_echo
          {
            # use 
            # kubectl logs -n kasten-io -l component=kanister --tail=10000 -f | ggrep -o -P '(?<=Elasticbps3echobegin).*(?=Elasticbps3echoend)' 
            # to grab the logs in kanisters 
            echo "Elasticbps3echobegin backup-action: $* Elasticbps3echoend"
          }
```

Reply:
you are right I mixed up I was using kubexec function at the beginning but could not keep up with this solution and switch to kubetask the dbg function is useless in this context I'm going to change that

Reply:
now the function bp_dbg is doing echo (as bp_echo) only to display repo information if debug=1.